forge-game/src/main/java/forge/game/card/CardState.java

```diff
@@ -452,6 +461,20 @@ public FCollectionView<ReplacementEffect> getReplacementEffects() {
             }
             result.add(loyaltyRep);
         }
+        if (getTypeWithChanges().isBattle()) {
+            if (defenseRep == null) {
+                defenseRep = CardFactoryUtil.makeEtbCounter("etbCounter:DEFENSE:" + this.baseDefense, this, true);
```

Comment:
@Hanmac Any idea why this would cause an infinite loop in replacement handling?

Reply:
Is it this one?

```
PatternSyntaxException: Stack overflow during pattern compilation near index 4 ((?<=\.)|(?=\.)) ^ at java.util.regex.Pattern.error(Pattern.java:2045) at forge.game.replacement.ReplacementHandler.run(ReplacementHandler.java:214)
```

I don't know yet, I need time to debug it myself